Common/src/Pollster/CreateLargeTaskProcessor.cs

```diff
@@ -0,0 +1,188 @@
+
+                                      await asyncPipe_.WriteAsync(new ProcessRequest
+                                                                  {
+                                                                    CreateTask = new ProcessRequest.Types.CreateTask
+                                                                                 {
+                                                                                   Reply = new CreateTaskReply
+                                                                                           {
+                                                                                             Successfull = new Empty(),
+                                                                                           },
+                                                                                   ReplyId = requestId_,
+                                                                                 },
+                                                                  })
+                                                      .ConfigureAwait(false);
+                                   },
+                                   cancellationToken);
+        break;
+      case ProcessReply.Types.CreateLargeTaskRequest.TypeOneofCase.InitTask:
+
+        switch (processReply.CreateLargeTask.InitTask.TypeCase)
+        {
+          case InitTaskRequest.TypeOneofCase.Header:
+            fsm_.AddHeader();
+            payloadsChannel_ = Channel.CreateUnbounded<ReadOnlyMemory<byte>>(new UnboundedChannelOptions
+                                                                             {
+                                                                               SingleWriter = true,
+                                                                               SingleReader = true,
+                                                                             });
+
+            await taskRequestsChannel_.Writer.WriteAsync(new TaskRequest(processReply.CreateLargeTask.InitTask.Header.Id,
+                                                                         processReply.CreateLargeTask.InitTask.Header.ExpectedOutputKeys,
+                                                                         processReply.CreateLargeTask.InitTask.Header.DataDependencies,
+                                                                         payloadsChannel_.Reader.ReadAllAsync(cancellationToken)),
+                                                         cancellationToken)
+                                      .ConfigureAwait(false);
+            
+
+            break;
+          case InitTaskRequest.TypeOneofCase.LastTask:
+            fsm_.CompleteRequest();
+            taskRequestsChannel_.Writer.Complete();
+            break;
+          case InitTaskRequest.TypeOneofCase.None:
+          default:
```

Comment:
It is probably the syntax but why do you need the line above with no treatment and the default after?

Reply:
The editor wants to see all the cases possible in a switch, so here they are

Reply:
Yes. It's a good way to find places where you do not handle new cases if you extends the enum.